src/main/java/mate/academy/springboot/datajpa/repository/CategoryRepo.java

```diff
@@ -0,0 +1,9 @@
+package mate.academy.springboot.datajpa.repository;
+
+import mate.academy.springboot.datajpa.model.Category;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface CategoryRepo extends JpaRepository<Category, Long> {
```

Comment:
```suggestion
public interface CategoryRepository extends JpaRepository<Category, Long> {
```
please fix in all places